apps/dashboard/pages/api/v1/github/verify.ts

```diff
@@ -0,0 +1,211 @@
+import { Buffer } from "node:buffer";
+import crypto from "node:crypto";
+import type { Readable } from "node:stream";
+import { db } from "@/lib/db";
+import { env } from "@/lib/env";
+import { clerkClient } from "@clerk/nextjs";
+import { sha256 } from "@unkey/hash";
+import { Resend } from "@unkey/resend";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+  runtime: "nodejs",
+};
+
+type Key = {
+  key_identifier: string;
+  key: string;
+  is_current: boolean;
+};
+
+// Needs to be tested when Github is live
+const verifyGitSignature = async (
+  payload: string,
+  signature: string,
+  keyId: string,
+  GITHUB_KEYS_URI: string,
+) => {
+  const gitHub = await fetch(GITHUB_KEYS_URI);
+  if (!gitHub.ok) {
+    console.error("Github verify error");
```

Comment:
log the error or at least status as well. This is impossible to debug right now
```suggestion
    console.error("Github verify error", gitHub.status, await gitHub.text());
```